pages/docs/concepts/class.en.mdx

```diff
@@ -265,3 +265,39 @@ fn accept_class_mut(engine: &mut QueryEngine) {
 export function acceptClass(engine: QueryEngine): void
 export function acceptClassMut(engine: QueryEngine): void
 ```
+
+## Get struct instances from Class instances
+
+In some cases you want to get struct instances from Class instances, you can use `FromNapiValue::from_unknown` like below:
+
+```rust filename="lib.rs"
+use napi::{bindgen_prelude::FromNapiValue, Result, JsObject};
+
+// A complex struct which cannot be exposed to JavaScript directly.
+#[napi]
+struct QueryEngine {
+  num: i32,
+}
+
+#[napi]
+impl QueryEngine {
+  #[napi(constructor)]
+  pub fn new() -> Result<Self> {
+    Ok(Self {
+      num: 42,
+    })
+  }
+}
+
+#[napi]
+pub fn unwrap(obj: JsObject) -> Result<()> {
```

Comment:
I think  `&mut QueryEngine` is a valid parameter

Reply:
Yeah, actually I want a generic function to unwrap `JsObject` without requiring a  binding function (i.e. unwrapping in pure rust functions).

Reply:
I'm not sure will other guys need this.  Currently,  we can use `FromNapiValue` though it's not documented. 

Or maybe we don't need document it as this might be not recommended. Then I think we can have regression tests for  `FromNapiValue` to make sure it won't be changed unexpectedly and other developers like me could use it safely.

Reply:
I have changed the example and description.

Anyway, I think the consideration here is should the trait `FromNapiValue` be public (it is, in fact) so that I don't need worry about using it.